wrappers/PhyML/README.md

```diff
@@ -0,0 +1,65 @@
+# **PhyML Galaxy Wrapper** <br>
+This repository contains a Galaxy tool wrapper for the PhyML program, which estimates the maximum likelihood phylogenies from alignments of nucleotide or amino acid sequences. <br>
+PhyML has been developed within the PhyML Development Team at ATGC Montpellier Bioinformatics platform. (*http://www.atgc-montpellier.fr/phyml/*)
+
+The reference for PhyML is: <br>
+	<pre>*New Alogirthms and Methods to Estimate Maximum-Likelihood Phylogenies: Assessing the Performance of PhyML* 
+Guindon S., Dufayard J.F., Lefort V., Anisimova M., Hordijk W., & Gascuel O. (2010). 3.0. 
+System Biology, 59(3):307-21. </pre>
+
+The PhyML Tool Suite repository: <br>
+<pre>* Provides PhyML wrapper for the PhyML tool 
+* Downloads and installs PhyML on the Linux operating system  </pre>
+
+Requirements: <br>
+<pre>* Linux 
+* Virtual Machine 
+* Galaxy
+* Galaxy Tool Shed </pre>
+
+
+### **Automated Installation** <br>
+Installation via the Galaxy Tool Shed will take care of installing the tool wrapper, and the phyml program. <br>
+	1. Start your virtual environment <br>
+	2. Create a category on your local Tool Shed called PhyML <br>
+	3. In that category, you need to create two repositories on your local Tool Shed: <br>
+		<pre>Name: phyml, Type: Unristricted Repository 
+Name: package_phyml_3_1, Type: Tool Dependency Repository </pre>
+
+</pre>	4. For the package_phyml_3_1 repository, you need to upload the file tool_dependencies.xml (located in wrappers/PhyML/package_phyml_3_1). This file wile will take care of installing PhyML. <br>
+	5. For the phyml repository, you need to upload two files called phyml_wrapper.xml and tool_dependencies.xml  (located in wrappers/PhyML/phyml). But before you upload these files, some changes must be made for the 	 wrappers/PhyML/phyml/tool_dependencies.xml. <br>
+	<pre>*Change the toolshed attribute to match your hostname: 
+		&lt;repository name="package_phyml_3_1 owner="galaxyuser" toolshed="http://yourmachinename.agr.gc.ca:9009"  ... >
+*Change the changeset_revision to match the revision of the package_phyml_3_1 repository
+		&lt;repository name="package_phyml_3_1 owner="galaxyuser" toolshed="http://yourmachinename.agr.gc.ca:9009" changeset_revision="revision of package_phyml_3_1" > </pre>
+
+</pre> 6. Go to your local Galaxy instance, click on Admin, then on Search and Browse Tool Sheds. Next, click on PhyML, and you can now install PhyML and run the tool.
```

Comment:
Break it up into 2 steps? Install PhyML and run the tools needs a bit more text. I.e. Install should mention that you only need to install phyml_wrapper and the package will be installed automatically as a dependency.